packages/paste-website/src/components/customization-landing-page/ReactCard.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+import {Paragraph} from '@twilio-paste/paragraph';
+import {ProductFlexIcon} from '@twilio-paste/icons/esm/ProductFlexIcon';
+
+import {ProjectCard, ProjectCardBody, ProjectHeading, ProjectLogo} from './ProjectCard';
+
+export const ReactCard: React.FC = () => {
+  return (
+    <ProjectCard>
+      <ProjectCardBody>
+        <ProjectLogo>
```

Comment:
Question/NIT: Is this somewhere we could use media object?

Reply:
I'm not sure I follow where? There's nothing to the right on this Logo.